Sources/App/Core/RSS.swift

```diff
@@ -86,26 +80,20 @@ extension RecentPackage {
 
 extension RecentRelease {
     var rssItem: Node<RSS.ChannelContext> {
-        let link = SiteURL.package(.value(repositoryOwner),
-                                   .value(repositoryName),
-                                   .none).absoluteURL(anchor: version)
+        let packageUrl = SiteURL.package(.value(repositoryOwner),
+                                         .value(repositoryName),
+                                         .none).absoluteURL()
+        let link = releaseUrl ?? packageUrl
         return .item(
             .guid(.text(link), .isPermaLink(true)),
             .title("\(packageName) - \(version)"),
             .link(link),
             .pubDate(releasedAt, timeZone: .utc),
             .description(
-                .p(
-                    .a(
-                        .href(link),
-                        .text(packageName)
-                    ),
-                    .element(named: "small", text: " – \(version)")
-                ),
```

Comment:
This was the redundant part of the RSS item. The link is already in the `.link(link)` item above (and it pointed to the SPI package).